integration/copy_test.go

```diff
@@ -103,13 +103,20 @@ func (s *CopySuite) TestCopyWithManifestList(c *check.C) {
 	assertSkopeoSucceeds(c, "", "copy", knownListImage, "dir:"+dir)
 }
 
-func (s *CopySuite) TestCopyAllWithManifestList(c *check.C) {
+func (s *CopySuite) TestCopyAllFlagWithManifestList(c *check.C) {
 	dir, err := ioutil.TempDir("", "copy-all-manifest-list")
 	c.Assert(err, check.IsNil)
 	defer os.RemoveAll(dir)
 	assertSkopeoSucceeds(c, "", "copy", "--all", knownListImage, "dir:"+dir)
 }
 
+func (s *CopySuite) TestCopyAllWithManifestList(c *check.C) {
```

Comment:
BTW I’m not sure we need this; remote image fetches are somewhat costly, and this is just a minimal test that the command doesn’t report failure without any verification of the contents.

There is a command with `--all` above, and `--multi-arch=all` elsewhere, so that’s probably sufficient.

(Do leave the new `TestCopyNoneWithManifestList` test around, it’s much cheaper and it’s unique.)

Reply:
Agreed on the extra test, when I first added it I didn't realise how expensive they are.